mcs/class/System.Private.CoreLib/System/Attribute.cs

```diff
@@ -6,6 +6,14 @@ partial class Attribute
 	{
 		static Attribute GetAttr (ICustomAttributeProvider element, Type attributeType, bool inherit)
 		{
+			if (attributeType == null)
+				throw new ArgumentNullException (nameof (attributeType));
+			
+			if (!attributeType.IsSubclassOf (typeof (Attribute)) &&
+			    attributeType != typeof (Attribute) &&
+			    attributeType != typeof (MonoCustomAttrs))
```

Comment:
When exactly is it `typeof (MonoCustomAttrs)`?

Reply:
https://github.com/mono/mono/blob/b1af4c2726cdc211971efacee7d66a6e447336d2/mcs/class/corlib/System/MonoCustomAttrs.cs#L298 and one more place somewhere, I guess it's used as a marker